docs/user-guide/persistent-volumes/index.md

```diff
@@ -534,6 +535,33 @@ parameters:
 * `location`: Azure storage account location. Default is empty.
 * `storageAccount`: Azure storage account name. If storage account is not provided, all storage accounts associated with the resource group are searched to find one that matches `skuName` and `location`. If storage account is provided, it must reside in the same resource group as the cluster, and `skuName` and `location` are ignored.
 
+#### ScaleIO
+```yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
```

Comment:
Please use `storage.k8s.io/v1`, I'm waiting for https://github.com/kubernetes/kubernetes.github.io/pull/2686 to get merged.